addons/website/static/src/js/content/menu.js

```diff
@@ -483,7 +483,9 @@ publicWidget.registry.FadeOutHeader = BaseDisappearingHeader.extend({
  * Note: this works well with the affixMenu... by chance (menuDirection is
  * called after alphabetically).
  *
- * @todo check bootstrap v4: maybe handled automatically now ?
+ * @todo For the moment, dynamic positioning of dropdown elements is explicitly
+ * disabled by Bootstrap on dropdowns that are in the navbar. Check in future
+ * versions of bootstrap if this limitation disappears.
```

Comment:
As explained in the Bootstrap documentation:
> Popper isn’t used to position dropdowns in navbars though as dynamic positioning isn’t required.

(Popper is the lib to position elements in BS) 
Strange they did that :thinking: 

I think that we could get rid of this `menuDirection` widget with a little fix of the Bootstrap dropdown library. Like this:
Replace that
```
_detectNavbar() {
      return this._element.closest(`.${CLASS_NAME_NAVBAR}`) !== null;
}
```
By that
```
_detectNavbar() {
      return false;
}
```
In a proper way off course and in master!

What do you think @qsm-odoo ?

Reply:
> Popper isn’t used to position dropdowns in navbars though as dynamic positioning isn’t required.

Strange indeed, I guess it will be worth changing that by patching the lib (if you can do it by patching the method from outside of the file).